core/src/main/java/hivemall/evaluation/GradedResponsesMeasures.java

```diff
@@ -54,4 +54,23 @@ public static double DCG(final List<Double> topRelScoreList, final int recommend
         return dcg;
     }
 
+
+    /**
+     * Computes Reciprocal HitRank (RHR)
+     *
+     * @param recommendList predicted item list order by score
+     * @param truthList gruond truth item list order by rank
+     * @param recommendSize the number of positive items
+     * @return RHR
+     */
+    public static double RHR(final List<?> recommendList, final List<?> truthList,
```

Comment:
@nzw0301 (cc: @takuti )

This is not graded response but binary response.

`ReciprocalRank` is already implemented in `BinaryResponseMeasures` and  `MRRUDAF` implements [Mean reciprocal rank](https://en.wikipedia.org/wiki/Mean_reciprocal_rank).

ARHRUDAF is duplicate to MRRUDAF and MRR is more generic name of ARHR.

[Average reciprocal hit-rate is also referred to as the mean reciprocal rank (MRR)](https://books.google.co.jp/books?id=GKjWCwAAQBAJ&pg=PA246&lpg=PA246&dq=%22average+reciprocal+hit-rate%22+MRR&source=bl&ots=Tq_wVfI-QU&sig=Hbyh-R1Sm7M26pZbHPjuTfyXXqI&hl=ja&sa=X&ved=0ahUKEwie1MD0prPWAhWMW7wKHVHzD88Q6AEIJzAA#v=onepage&q=%22average%20reciprocal%20hit-rate%22%20MRR&f=false)